CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticViewTopComponent.java

```diff
@@ -208,6 +208,7 @@ protected void edit(final GraphWriteMethods graph, final PluginInteraction inter
                                 graph.setBooleanValue(transactionSelectedAttribute, transactionId, elementIds.contains(transactionId));
                             }
                             break;
+                        default:
```

Comment:
Although it will fall through and likely work, isn't it good practice to use break after default?

Reply:
For the reasons mention in [this answer](https://softwareengineering.stackexchange.com/a/201786) I'd agree with adding a break at the end; for consistency - i.e. The Principle of Least Surprise

Reply:
This has now been done. I applied to a few extra files that also didn't have breaks in the default case